ansible/playbooks/de-deploy-services.yaml

```diff
@@ -299,41 +316,6 @@
       when: not parasitic
       msg: ":heavy_check_mark: Done deploying dewey"
 
-- name: Redeploy de-ui
-  hosts: ui:&systems
-  become: true
-  gather_facts: false
-  tags:
-    - ui
-    - uncolored
-  roles:
-    - role: util-notify-chat
-      msg: "Deploying UI"
-    - role: de-deploy-service
-      has_iplant_data: true
-      deploy_use_color: false
-      service_name: "{{de.compose_service}}"
-    - role: util-notify-chat
-      msg: ":heavy_check_mark: Done deploying UI"
-
-- name: Redeploy de-ui-nginx
-  hosts: ui:&systems
-  become: true
-  gather_facts: false
-  tags:
-    - ui
-    - uncolored
-    - de-ui-nginx
-  roles:
-    - role: util-notify-chat
-      msg: "Deploying UI nginx"
```

Comment:
minor side note: I removed these notifications in moving them to de-deploy-service-nginx, since the other things there don't have notifications. It might make sense to add them back to the whole file at some point in the future